src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int bound = 100;
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor().name(), random.nextInt(bound));
```

Comment:
let's declare colorSupplier the same way as random. collorSupplier and random objects are your dependencies and should be initialized in class fields